dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/udp/UDPConfigurationImpl.java

```diff
@@ -0,0 +1,467 @@
+                    key.startsWith("config.") ||
+                    key.startsWith("objectClass") ||
+                    key.startsWith("parentPid")) {
+                    // skip osgi standard properties
+                    continue;
+                }
+
+                if (isInboundChannel()) {
+                    if (key.equalsIgnoreCase(UDPConfigConstants.HOST_NAME)) {
+                        setHostname((String) value);
+                        continue;
+                    }
+                    if (key.equalsIgnoreCase(UDPConfigConstants.PORT)) {
+                        setPort(Integer.parseInt((String) value));
+                        continue;
+                    }
+                    if (key.equalsIgnoreCase(UDPConfigConstants.ENDPOINT_NAME)) {
+                        // This parameter only has to be present if WAS is running,
+                        // It is used by the Channel Framework later.
+                        // Don't throw a warning and move on.
+                        continue;
+                    }
+                    if (key.equalsIgnoreCase(UDPConfigConstants.SEND_BUFF_SIZE)) {
+                        setSendBufferSize(Integer.parseInt((String) value));
+                        continue;
+                    }
+                    if (key.equalsIgnoreCase("externalName")) {
+                        this.setExternalName((String) value);
+                        continue;
+                    }
+                }
+                if (key.equalsIgnoreCase(UDPConfigConstants.SEND_BUFF_SIZE)) {
+                    setSendBufferSize(Integer.parseInt((String) value));
+                    continue;
+                }
+                if (key.equalsIgnoreCase(UDPConfigConstants.RCV_BUFF_SIZE)) {
+                    setReceiveBufferSize(Integer.parseInt((String) value));
+                    continue;
+                }
+                if (key.equalsIgnoreCase(UDPConfigConstants.CHANNEL_RCV_BUFF_SIZE)) {
+                    setChannelReceiveBufferSize(Integer.parseInt((String) value));
+                    continue;
+                }
+                if (key.equalsIgnoreCase(UDPConfigConstants.ADDR_EXC_LIST)) {
+                    if (value instanceof String) {
+                        this.addressExcludeList = convertToArray((String) value);
+                    } else {
+                        this.addressExcludeList = (String[]) value;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(UDPConfigConstants.ADDR_INC_LIST)) {
+                    if (value instanceof String) {
+                        this.addressIncludeList = convertToArray((String) value);
+                    } else {
+                        this.addressIncludeList = (String[]) value;
+                    }
+                    continue;
+                }
+
+//                if (key.equalsIgnoreCase(ChannelFrameworkConstants.CHAIN_DATA_KEY)
```

Comment:
'this channel' seems odd - does it mean this UDP channel impl does not make us if it?